### PR TITLE
⚠️ CONFLICT! Lineage pull request for: skeleton

### PR DESCRIPTION
**Pull request by cisagovbot**
# Lineage Pull Request: CONFLICT #

[Lineage] has created this pull request to incorporate new changes found in an
upstream repository:

Upstream repository: [`https://github.com/cisagov/skeleton-docker.git`](https://github.com/cisagov/skeleton-docker.git)
Remote branch: `HEAD`

Check the changes in this pull request to ensure they won't cause issues with
your project.

The `lineage/skeleton` branch has **one or more unresolved merge conflicts**
that you must resolve before merging this pull request!

## How to resolve the conflicts ##

1. Take ownership of this pull request by removing any other assignees.

1. Clone the repository locally, and reapply the merge:

    ```console
    git clone git@github.com:cisagov/postfix-docker.git postfix-docker
    cd postfix-docker
    git remote add skeleton https://github.com/cisagov/skeleton-docker.git
    git remote set-url --push skeleton no_push
    git switch develop
    git checkout -b lineage/skeleton --track origin/develop
    git pull skeleton HEAD
    git status
    ```

1. Review the changes displayed by the `status` command.  Fix any conflicts and
   possibly incorrect auto-merges.

1. After resolving each of the conflicts, `add` your changes to the
   branch, `commit`, and `push` your changes:

    ```console
    git add .github/workflows/build.yml .github/workflows/release.yml README.md tests/container_test.py 
    git commit
    git push --force --set-upstream origin lineage/skeleton
    ```

    Note that you may _append_ to the default merge commit message
    that git creates for you, but *please do not delete the existing
    content*.  It provides useful information about the merge that is
    being performed.

1. Wait for all the automated tests to pass.

1. Check the "Everything is cool" checkbox below:

    - [x] ✌️ The conflicts in this pull request have been resolved.

1. Mark this draft pull request "Ready for review".

------------

**Note:** *You are seeing this because one of this repository's maintainers has
configured [Lineage] to open pull requests.*

For more information:

🛠 [Lineage] configurations for this project are stored in `.github/lineage.yml`

📚 [Read more about Lineage][Lineage]

[//]: # (lineage:metadata:{&quot;slayed&quot;:true})
[Lineage]: https://github.com/cisagov/action-lineage/ "Lineage GitHub Action"

### CODE REVIEW

**Review by felddy — APPROVED** [review]
🎵 I hope you know that this will go down on your permanent record... 📁

**Review by jsf9k — APPROVED** [review]
Go forth and educate the masses on the harmonious beauty of the solidus!

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -3,17 +3,44 @@ name: build
...
+        required: true
+        default: dispatch
 
 env:
+  BUILDX_CACHE_DIR: ~/.cache/buildx
+<<<<<<< HEAD
   IMAGE_NAME: cisagov/postfix
+=======
+  IMAGE_NAME: cisagov/example
+>>>>>>> 9e993fbb4bbad507742670c91d98806190f37f0f
```
Comment (dav3r) [inline .github/workflows/build.yml]: Some merge markers were missed here.

File: README.md [hunk 2]
```diff
@@ -6,8 +6,15 @@
...
-![MicroBadger Size](https://img.shields.io/microbadger/image-size/cisagov/postfix.svg)
+<<<<<<< HEAD
+[![Docker Pulls](https://img.shields.io/docker/pulls/cisagov/postfix)](https://hub.docker.com/r/cisagov/postfix)
+[![Docker Image Size (latest by date)](https://img.shields.io/docker/image-size/cisagov/postfix)](https://hub.docker.com/r/cisagov/postfix)
+[![Platforms](https://img.shields.io/badge/platforms-amd64%20%7C%20arm%2Fv6%20%7C%20arm%2Fv7%20%7C%20arm64%20%7C%20ppc64le%20%7C%20s390x-blue)](https://hub.docker.com/r/cisagov/postfix/tags)
+=======
+[![Docker Pulls](https://img.shields.io/docker/pulls/cisagov/example)](https://hub.docker.com/r/cisagov/example)
+[![Docker Image Size (latest by date)](https://img.shields.io/docker/image-size/cisagov/example)](https://hub.docker.com/r/cisagov/example)
+[![Platforms](https://img.shields.io/badge/platforms-amd64%20%7C%20arm%2Fv6%20%7C%20arm%2Fv7%20%7C%20arm64%20%7C%20ppc64le%20%7C%20s390x-blue)](https://hub.docker.com/r/cisagov/skeleton-docker/tags)
+>>>>>>> 9e993fbb4bbad507742670c91d98806190f37f0f
```
Comment (dav3r) [inline README.md]: More missed mergies here....

File: tests/container_test.py [hunk 3]
```diff
@@ -177,5 +177,9 @@ def test_container_version_label_matches(main_container):
         exec(f.read(), pkg_vars)  # nosec
     project_version = pkg_vars["__version__"]
     assert (
-        main_container.labels["version"] == project_version
+<<<<<<< HEAD
+        main_container.labels["org.opencontainers.image.version"] == project_version
+=======
+        version_container.labels["org.opencontainers.image.version"] == project_version
+>>>>>>> 9e993fbb4bbad507742670c91d98806190f37f0f
```
Comment (dav3r) [inline tests/container_test.py]: And here too.

**Review by dav3r — CHANGES_REQUESTED** [review]
Pause, take a deep breath, then try again.  🤕

**Review by jsf9k — APPROVED** [review]
I still love it!

File: tests/container_test.py (hunk 3 above)
Comment (felddy) [reply tests/container_test.py]: Good catch.   Not sure how I did this, but everything on my local repo is good.  Re-force pushing...

File: .github/workflows/build.yml (hunk 1 above)
Comment (felddy) [reply .github/workflows/build.yml]: How did this even run?!  The lights were all green.  
Sometimes I amaze myself.

**Review by dav3r — APPROVED** [review]
Now we're cooking! 🍝

**Pull request merged**